src/APP/pages/aboutUs/sections/LeadershipSection.jsx

```diff
@@ -1,72 +1,73 @@
-import JamesOtieno from "../../../../assets/images/aboutPage/leadership/james-otieno.png";
-import HudsonObai from "../../../../assets/images/aboutPage/leadership/hudson-obai.png";
-import IanMugenya from "../../../../assets/images/aboutPage/leadership/ian-mugenya.png";
-import MarlynMayienga from "../../../../assets/images/aboutPage/leadership/marlyn-mayienga.png";
-import SharonJebitok from "../../../../assets/images/aboutPage/leadership/sharon-jebitok.png";
-import WaithakaWaweru from "../../../../assets/images/aboutPage/leadership/waithaka-waweru.png";
-import JumaLawrence from "../../../../assets/images/aboutPage/leadership/juma-lawrence.png";
-import Caroussel from "../../../components/Caroussel";
-import BuildingCommunity from "../../../../assets/images/aboutPage/building-community.png";
-
-const LeadershipData = [
-  {
-    name: "Fred Ouko",
-    title: "Founder",
-    image: FredOuko,
-  },
-  {
-    name: "Catherine Kiiru",
-    title: "Dev Relations & Opensource Programs",
-    image: CatherineKiiru,
-  },
-  {
-    name: "James Otieno",
-    title: "Community Manager",
-    image: JamesOtieno,
-  },
-  {
-    name: "Hudson Obai",
-    title: "Head of Engineering",
-    image: HudsonObai,
-  },
-  {
-    name: "Ian Mugenya",
-    title: "Mentorship Program Lead",
-    image: IanMugenya,
-  },
-  {
-    name: "Marlyn Mayienga",
-    title: "Program Manager",
-    image: MarlynMayienga,
-  },
-  {
-    name: "Sharon Jebitok",
-    title: "Head of Chapters and Volunteers",
-    image: SharonJebitok,
-  },
-  {
-    name: "Waithaka Waweru",
-    title: "Dev Relations and Events",
-    image: WaithakaWaweru,
-  },
-  {
-    name: "Juma Lawrence",
-    title: "Head of Communication",
-    image: JumaLawrence,
-  },
-];
+import { Fragment, useState, useRef } from "react";
+import { Dialog, Transition } from "@headlessui/react";
+import emailjs from "@emailjs/browser";
+
+import { Caroussel } from "../../../components";
+import { closeIcon } from "../../../../assets/images/icons";
+import { LeadershipData } from "../data";
+import { buildComm } from "../../../../assets/images/aboutPage";
 
 function LeadershipSection() {
+  const [isOpen, setIsOpen] = useState(false);
+  const [loading, setLoading] = useState(false);
+
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
+  const [message, setMessage] = useState("");
+
+  const formRef = useRef();
+
+  function closeModal() {
+    setIsOpen(false);
+  }
+
+  function openModal() {
+    setIsOpen(true);
+  }
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setLoading(true);
+
+    emailjs
+      .send(
+        "service_rvx6y2h",
+        "template_8fq0b3l",
+        {
+          from_name: name,
+          to_name: "SYT Admin",
+          from_email: email,
+          to_email: "felix5olali@gmail.com",
+          message,
+        },
+        "maz5XYDLGqV1Eo4h7"
```

Comment:
Could you please convert the service id, template id and public key to env variables?

Reply:
Yes. We'll do this before shipping